### PR TITLE
add missing state for paused case

### PR DESCRIPTION
**Pull request by jsdosa**
The condition of resume is needed.

Signed-off-by: jaesick.shin <jaesick.shin@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [f395d206bf379091ccf8785c504c67899c87e3c6] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [f395d206bf379091ccf8785c504c67899c87e3c6] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [f395d206bf379091ccf8785c504c67899c87e3c6] -

**Comment by seinfra** [comment]
Target : [ca565264418bb9803946351f086a37e845955d5d] -

**Comment by seinfra** [comment]
Target : [ca565264418bb9803946351f086a37e845955d5d] -

File: framework/src/media/MediaRecorderImpl.cpp [hunk 1]
```diff
@@ -228,9 +228,10 @@ void MediaRecorderImpl::startRecorder()
 {
 	medvdbg("MediaRecorderImpl::startRecorder()\n");
 
-	if (mCurState != RECORDER_STATE_READY) {
+	if (mCurState != RECORDER_STATE_READY && mCurState != RECORDER_STATE_PAUSED) {
 		notifyObserver(OBSERVER_COMMAND_ERROR);
-		meddbg("MediaRecorderImpl::startRecorder() - (mCurState != RECORDER_STATE_READY)\n");
+		meddbg("MediaRecorderImpl::startRecorder() - " \
+			"mCurState != RECORDER_STATE_READY && mCurState != RECORDER_STATE_PAUSED\n");
```
Comment (Taejun-Kwon) [inline framework/src/media/MediaRecorderImpl.cpp]: In this case, we'd better print out mCurstate with %d

**Comment by seinfra** [comment]
Target : [63b0f592da6b2b149af08c10e222cfeb58f4cad4] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [63b0f592da6b2b149af08c10e222cfeb58f4cad4] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [63b0f592da6b2b149af08c10e222cfeb58f4cad4] -

File: framework/src/media/MediaRecorderImpl.cpp (hunk 1 above)
Comment (jsdosa) [reply framework/src/media/MediaRecorderImpl.cpp]: yes, you right,
after pre 2.0 i will change the all logs.

**Pull request merged**